article1_be/article1-be/src/main/java/article1be/outfit/service/OutfitSelectionService.java

```diff
@@ -0,0 +1,60 @@
+package article1be.outfit.service;
+
+import article1be.outfit.dto.OutfitSelectionRequestDTO;
+import article1be.outfit.entity.SelectRecord;
+import article1be.outfit.entity.SelectOutfit;
+import article1be.outfit.repository.SelectOutfitRepository;
+import article1be.outfit.repository.SelectRecordRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+@RequiredArgsConstructor
+@Service
+public class OutfitSelectionService {
+    private final SelectRecordRepository selectRecordRepository;
+    private final SelectOutfitRepository selectOutfitRepository;
+
+    @Transactional
+    public void saveSelectedOutfits(OutfitSelectionRequestDTO requestDTO) {
+        // SelectRecord 생성 시 customDate 포함
+        SelectRecord selectRecord = SelectRecord.create(
+                requestDTO.getUserSeq(),
+                requestDTO.getSituationSeq(),
+                LocalDateTime.now(),
+                requestDTO.getCustomDate(),
+                requestDTO.getCustomLocation(),
+                requestDTO.getWeatherCode(),
+                requestDTO.getHighTemp(),
+                requestDTO.getLowTemp(),
+                requestDTO.getDailyTemp(),
+                requestDTO.getCurTemp(),
+                requestDTO.getPrecipitation()
+        );
```

Comment:
SelectRecord의 create를 이곳에서만 사용한다면
```suggestion
public static SelectRecord create(OutfitSelectionRequestDTO requestDTO){}
// 이것과 아래를 함께 사용
SelectRecord.create(requestDTO)
```
이렇게 사용하면 좋을거 같네욤